doc/src/bdist_dmg.rst

```diff
@@ -19,6 +19,67 @@ installation.
        image
    * - .. option:: silent (-s)
      - suppress all output except warnings
+   * - .. option:: format
+     - Format of the DMG disk image Default is UDZO
+   * - .. option:: filesystem
+     - Filesystem of the DMG disk image Default is HFS+
+   * - .. option:: size
+     - If defined, specifies the size of the filesystem within the image.
+       If this is not defined, cx_Freeze (and then dmgbuild) will attempt to
+       determine a reasonable size for the image. If you set this, you should
+       set it large enough to hold the files you intend to copy into the image.
+       The syntax is the same as for the -size argument to hdiutil, i.e. you
+       can use the suffixes `b`, `k`, `m`, `g`, `t`, `p` and `e` for bytes,
+       kilobytes,megabytes, gigabytes, terabytes, exabytes and petabytes
+       respectively.
+   * - .. option:: background
+     - A rgb color in the form #3344ff, svg named color like goldenrod, a path
+       to an image, or the words 'builtin-arrow'. Default is None.
+   * - .. option:: show-status-bar
+     - Show the status bar in the Finder window. Default is False.
+   * - .. option:: show-tab-view
+     - Show the tab view in the Finder window. Default is False.
+   * - .. option:: show-path-bar
```

Comment:
Here in the docs, the options should use underline, ie, show_path_bar.

Reply:
Done in [`5b5edd3` (#2442)](https://github.com/marcelotduarte/cx_Freeze/pull/2442/commits/5b5edd32ff70788b6a14540731ffe52ecf8ea4c7)